### PR TITLE
Clarify install option in example

### PR DESCRIPTION
**Pull request by rodrigc**
Signed-off-by: Craig Rodrigues <rodrigc@crodrigues.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ jobs:
 
 ### Install by default
 
+If you want set up the `docker build` command as an alias to `docker buildx`:
```
Comment (crazy-max) [inline README.md:98]:
```suggestion
If you want set up the `docker build` command as an alias to `docker buildx build`:
```

**Comment by crazy-max** [comment]
> @crazy-max this line mentions https://github.com/docker/setup-buildx-action/blob/master/action.yml#L25
> 
> ```
> description: 'Sets up docker build command as an alias to docker buildx'
> ```
> 
> Does that need to be changed also?

Yes please

**Comment by rodrigc** [comment]
> 
> Yes please

OK, done.

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Pull request merged**